spec/jobs/reports/drop_off_report_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe Reports::DropOffReport do
+  let(:report_date) { Date.new(2023, 12, 12).in_time_zone('UTC') }
+
+  before do
+    allow(IdentityConfig.store).to receive(:s3_reports_enabled).and_return(true)
+    allow(IdentityConfig.store).to receive(:team_ursula_email).and_return('ada@example.com')
+  end
+
+  describe '#perform' do
+    it 'gets a CSV from the report maker, saves it to S3, and sends email to team' do # FAILED
+      reports =
+        Reporting::EmailableReport.new(
+          title: 'Drop Off Report',
+          table: [
+            ['Report Timeframe', '2023-12-10 00:00:00 UTC to 2023-12-10 23:59:59 UTC'],
+            ['Report Generated', '2023-12-11'],
+            ['Issuer', 'some:issuer'],
+            ['Metric', '# of Users'],
+            [],
+            ['Started IdV Verification', '78'],
+            ['Submitted welcome page', '75'],
+            ['Images uploaded', '73'],
+            [],
+            ['Workflow completed', '72'],
+            ['Workflow completed - Verified', '71'],
+            ['Workflow completed - Total Pending', '33'],
+            ['Workflow completed - GPO Pending', '23'],
+            ['Workflow completed - In-Person Pending', '55'],
+            ['Workflow completed - Fraud Review Pending', '34'],
+            [],
+            ['Successfully verified', '56'],
+            ['Successfully verified - Inline', '55'],
+            ['Successfully verified - GPO Code Entry', '25'],
+            ['Successfully verified - In Person', '25'],
+            ['Successfully verified - Passed Fraud Review', '15'],
+          ],
+          filename: 'drop_off_report',
+        )
+
+      report_maker = double(
+        Reporting::DropOffReport,
+        to_csv: 'I am a CSV, see',
+        identity_verification_emailable_report: reports,
+      )
+      allow(subject).to receive(:report_maker).and_return(report_maker)
```

Comment:
i'd pull these things into `let` and `before` blocks, something like this

```ruby
let(:report_maker) do
  double(
        Reporting::DropOffReport,
        to_csv: 'I am a CSV, see',
        identity_verification_emailable_report: [reports],
      )
end

before do
  allow(Reporting::DropOffReport).to receive(:new).and_return(report_maker)
end 
```

Reply:
As someone new to this project, the distinction between 
`Reporting::DropOffReport` and 
`Reports::DropOffReport` sure took me a few reads of the code to catch up on